src/Actions.js

```diff
@@ -35,7 +35,7 @@ export function create({ name, map }) {
   function action(value) {
     let err = null;
     try {
-      value = map(value);
+      value = map.bind(this)(value);
     } catch(e) {
```

Comment:
I misunderstood what you meant by "move the bind", though it was in retrospect quite clear. Updating...